dynamic_env_pytest_tests/tests/services/s3_gate/test_s3_gate.py

```diff
@@ -0,0 +1,559 @@
+import logging
+import os
+from random import choice, choices
+
+import allure
+import pytest
+from aws_cli_client import AwsCliClient
+from common import ASSETS_DIR
+from file_helper import (
+    generate_file,
+    generate_file_with_content,
+    get_file_content,
+    get_file_hash,
+    split_file,
+)
+from s3_helper import (
+    check_objects_in_bucket,
+    check_tags_by_bucket,
+    check_tags_by_object,
+    set_bucket_versioning,
+    try_to_get_objects_and_expect_error,
+    parametrize_clients
+)
+
+from pytest_tests.steps import s3_gate_bucket, s3_gate_object
+from s3.s3_gate_base import TestNeofsS3GateBase
+
+logger = logging.getLogger("NeoLogger")
+
+
+def pytest_generate_tests(metafunc):
+    parametrize_clients(metafunc)
+
+
+@allure.link("https://github.com/nspcc-dev/neofs-s3-gw#neofs-s3-gateway", name="neofs-s3-gateway")
```

Comment:
to me its too hard ref for this code space, i'd place it in some top-level header file